content/react.md

```diff
@@ -238,6 +238,73 @@ Although this `ListsShowPage` container is intended to be instantiated by the Re
 <ListsShowPage params={{id: '7'}}/>
 ```
 
+<h3 id="using-tracker-component">Using `Tracker.Component`</h3>
+
+Once you've run `npm i --save tracker-component`, you'll be able to create components with the `Tracker.Component` class, it allows you to create a [container component](https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0#.by86emv9b) which provides data to your presentational components.
```

Comment:
I'm pretty sure createContainer does unsubscribe... Do you have a repro? That would be a pretty huge oversight otherwise.